client/scripts/challenges/projecteuler/formatted/001_to_100/Problem 019: Counting Sundays.js

```diff
@@ -0,0 +1,61 @@
+
+/* eslint spaced-comment: 0 */
+/* eslint no-redeclare: 0 */
+/* eslint no-unused-vars: 0 */
+
+const assert = require('chai').assert;
+
+/// title: Problem 19: Counting Sundays
+/// type: project-euler
+
+/// categories:
+/// math
+
+/// difficulty: ?
+
+/// benchmark:
+//replaceWithActualFunctionHere;
+
+/// description:
+/// <div class="euler"><p class="euler__paragraph">You are given the following information, but you may prefer to do some research for yourself.</p>
+/// <p class="euler__paragraph">1 Jan 1900 was a Monday.</p>
+/// <p class="euler__paragraph">Thirty days has September,</p>
+/// <p class="euler__paragraph">April, June and November.</p>
+/// <p class="euler__paragraph">All the rest have thirty-one,</p>
+/// <p class="euler__paragraph">Saving February alone,</p>
+/// <p class="euler__paragraph">Which has twenty-eight, rain or shine.</p>
+/// <p class="euler__paragraph">And on leap years, twenty-nine.</p>
```

Comment:
I think it's better to use bullet points like in https://projecteuler.net/problem=19
so it's more readable.